blog/craftmanship/2024-10-25-tyranny-of-convenience/index.md

```diff
@@ -0,0 +1,91 @@
+Si la réponse est non, et particulièrement si la raison pour laquelle vous prenez cette décision est "par habitude" (pour ne pas avoir à transférer vos données/contacts, parce que vous avez l'habitude de l'UI, 
+parce que c'est ce qui est fait partout ailleurs dans le projet, parce que votre cloud provider propose une solution clef en main, parce que c'est inclu dans notre offre, parce que c'est un outil que vous maitrisez déjà...)
+alors vous ne prenez pas une décision rationnelle : vous vous pliez au marketing et à la tyrannie de la commodité.
+
+Prenez le temps d'envisager d'autres options, de regarder du côté de l'open source. Ne partez pas du principe que le plus évident et le plus rationnel.
+
+#### 2. Apprendre à repérer les situations à risques :
+Il est plus facile de lutter contre la tyrannie de la commodité avant d'en être victime. Certains concepts doivent nous mettre la puce à l'oreille au moment de choisir un outil :
+"Facile", "clef en main", "tout compris", "le plus utilisé". Ce genre de mot clef doit nous interroger. Soyons clair : il ne faut pas fuir la commodité par principe, mais être conscient de ses effets négatifs.
+"Facile" doit venir en bonus de "adapté". "Facile" n'est pas le synonyme d'"efficace". Et on sait que, de la mécanique automobile au développement d'application,
+il est souvent plus simple pour les entreprises fabriquant des outils de garder les utilisateurs·ices captifs·ves plutôt que de fournir les meilleurs outils et les plus adaptés. 
+Au moment de choisir un nouvel outil, ou de faire un choix technique, méfions nous des arguments de commodité : les décisions que nous prenons maintenant vont nous contraindre dans le futur : faisons en sorte que ce soit pour de bonnes raisons.
+
+#### 3. Éduquer notre entourage :
+Comme nous l'avons vu, la tyrannie de la commodité a des effets plus larges qu'à l'échelle d'un projet, d'un choix technique, ou de l'achat nouveau smartphone : elle appauvrit tout notre environnement technique,
+limite nos choix, et ralentit l'innovation. Mais si vous essayez demain de convaincre vos collègues de ne pas utiliser Google, ou de passer de iOS à Android, vous risquez de vous heurter à un mur.
+Les habitudes sont difficiles à changer et la tyrannie de la commodité a cela de pernicieux qu'elle fait apparaître toutes les alternatives comme stupides, irrationnelles, irréalisables.
+Parler de la tyrannie de la commodité et de ses effets délétères (en partageant cet article par exemple 😊) peut permettre de faire changer les mentalités, 
+en faisant prendre conscience à votre entourage et à vos collègues que ce qu'iels pensent être un choix dicté par leur rationalité est en réalité une contrainte,
+ou pire : une manipulation marketing. En cédant à cette facilité, iels limitent non seulement leurs options, mais celle de communauté entière.
+
```

Comment:
Il faudrait peut-être une conclusion avec une autre citation sur la commodité et nuancer un petit peu. À la lecture de ton article, j'ai (très subjectif) l'impression que TOUS nos choix sont par commodités. Dernière suggestion : on pourrait appuyer ton argumentaire sur le marketing par les noms de produits qui sont devenus des noms commun ou des verbes, alors qu'ils ne sont pas forcément les meilleurs produits. Scotch, ou Post-it (car on ne veut pas dire petite feuille de papier autoadhésive amovible), On prend un Uber parceque l'appli est simple et rapide, mais finalement appeler un chauffeur de taxi qui a du métier n'est-il pas mieux pour l'ipact économique local, la qualité du transport... ou autre exemple.